streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/master/graphmanager/GraphManager.java

```diff
@@ -0,0 +1,33 @@
+package org.ray.streaming.runtime.master.graphmanager;
+
+import java.io.Serializable;
+import org.ray.streaming.jobgraph.JobGraph;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionGraph;
+
+/**
+ * The central role of graph management in JobMaster.
```

Comment:
We should  explain what a `GraphManager` is here.

Reply:
Add more explanation here.